paymentsheet/src/test/java/com/stripe/android/paymentsheet/paymentdatacollection/cvcrecollection/CvcRecollectionScreenScreenshotTest.kt

```diff
@@ -51,4 +62,15 @@ class CvcRecollectionScreenScreenshotTest {
             )
         }
     }
+
+    @Test
+    fun testFilledTestModePaymentScreenDisplayMode() {
+        paparazziRule.snapshot {
```

Comment:
it looks like this is now equivalent to the `testEmptyPaymentSheetDisplayMode` test, do we still need both?